omas/machine_mappings/_common.py

```diff
@@ -9,6 +9,108 @@
 __support_files_cache__ = {}
 
 
+class D3DFitweight(dict):
+    """
+    OMFIT class to read DIII-D fitweight file
+    """
+
+    def __init__(self, filename):
+        r"""
+        OMFIT class to parse DIII-D device files
+
+        :param filename: filename
+
+        :param \**kw: arguments passed to __init__ of OMFITascii
+        """
+        self.filename = filename
+        self.load()
```

Comment:
You shouldn't do this because the following syntax is used to load these files:
```
def get_support_file(object_type, filename):
    """
    Cached loading of support files

    :param object_type: Typically a OMFIT class

    :param filename: filename of the support file to load
    """
    if filename not in __support_files_cache__:
        __support_files_cache__[filename] = object_type(filename)
        __support_files_cache__[filename].load()
    return __support_files_cache__[filename]
```

```suggestion
```